app/upgradev3x3/migrate.go

```diff
@@ -5,93 +5,20 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
-	govv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
-	gov "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
-	"github.com/gogo/protobuf/proto"
-
-	"github.com/umee-network/umee/v6/x/leverage/types"
 )
 
-// Implements Proposal Interface
-var _ gov.Content = &UpdateRegistryProposal{}
-
-func registerInterfaces(registry cdctypes.InterfaceRegistry) {
-	registry.RegisterImplementations(
-		(*gov.Content)(nil),
-		&UpdateRegistryProposal{},
-	)
-}
-
 type migrator struct {
 	gov govkeeper.Keeper
 }
 
 // Creates migration handler for gov leverage proposals to new the gov system
 // and MsgGovUpdateRegistry type.
 func Migrator(gk govkeeper.Keeper, registry cdctypes.InterfaceRegistry) module.MigrationHandler {
```

Comment:
Will we run into "unknown field: title" when attempting to load historic proposals for gov queries? Perhaps we need another migration similar to how this one worked. (Unless store load safely ignores unknown fields)